lib/v2/screens/app/components/account_under_recovery_screen.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/v2/components/flat_button_long.dart';
+import 'package:seeds/v2/screens/app/interactor/viewmodels/app_bloc.dart';
+import 'package:seeds/v2/screens/app/interactor/viewmodels/app_event.dart';
+
+class AccountUnderRecoveryScreen extends StatelessWidget {
```

Comment:
this entire screen UI is not completed. Functionality is in place, But the UI sucks. 
@gguij004  please fix this UI

Reply:
I will once this code is merge (: